test/config_file.c

```diff
@@ -943,6 +955,45 @@ extra-mount-opts=ro,noatime\n";
 	g_assert_cmpstr(slot->extra_mount_opts, ==, "ro,noatime");
 }
 
+static void config_file_extra_mkfs_opts(ConfigFileFixture *fixture,
+		gconstpointer user_data)
+{
+	g_autoptr(RaucConfig) config = NULL;
+	GError *ierror = NULL;
+	gboolean res;
+	g_autofree gchar* pathname = NULL;
+	RaucSlot *slot = NULL;
+
+	const gchar *cfg_file = "\
+[system]\n\
+compatible=FooCorp Super BarBazzer\n\
+bootloader=barebox\n\
+mountprefix=/mnt/myrauc/\n\
+activate-installed=false\n\
```

Comment:
This is a quite specific configuration and unrelated to the test, thus better remove it to make test purpose clearer.
```suggestion
```